### ISSUE TITLE
I'm in Jail

### ISSUE THREAD

**Issue by cubanito1388** [HomepointXI/Issues#105]
**Date & Time**: 

06/28/2018 5:49pm

**Client Version (use `/ver` in game):**

30180604_0

**Character Name:**
Destinyguardian

**Discord Name (if different from in game):**

Destinyguardiancu#6773

**Nation:**

San d'oria

**Job(level)/Sub Job(level):**

THF99/DNC49

**NPC or Monster or item Name:**

NA

**Zone name:** 

**Coordinates (use `!where`):**

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):**

**Steps to reproduce:**

**Any additional information:**
I used /debuff command and without knowing it was illegal to use it. I did learned my less.

**Comment by EpicTaru** [comment]
This happens from using tools or addons that try to strip off status effects that you normally cannot.

Each time that happens the code will re-jail, even if you are already in jail. Please disable any such tools and remove them and any Windower or Ashita scripts or macros that would attempt to automatically remove such effects (like weakness) so that you don't auto-jail yourself again, and you will be released at the earliest convenience of a GM or admin. Hopefully you will never see yourself in jail again.

tl;dr
A GM or Admin will be with you when one gets the chance to.

**Comment by cubanito1388** [comment]
Thanks for the info bro.

**Comment by cubanito1388** [comment]
all scripts has been removed from ashita as we speak. Sorry for the inconvenience.

**Comment by cubanito1388** [comment]
all addon too

**Comment by EpicTaru** [comment]
Released yesterday by Immo.
